experimental/dispatch_profiler/options.py

```diff
@@ -148,6 +161,17 @@ def add_performance_report_arguments(parser):
 def parse_generator_arguments(parser):
   """Adds and parse all the arguments for the *generator.py* script."""
   add_typical_arguments(parser)
+  add_matmul_arguments(parser)
+  add_iree_compile_arguments(parser)
```

Comment:
I'm curious why the generator needs compile arguments?

Reply:
generator and iree_compile has a few shared arguments, e.g., 

```
iree_compile_parser.add_argument(
                      "--iree-hal-target-backends", "--device", \
                      dest="device", default="cuda", \
                      help="Target backends for executable compilation. ", \
                      choices=["cuda", "vulkan", "cpu"])
```

For now, we are only using cuda and it triggers `CudaMatmulGenerator`. Matmuls running Cuda can be fairely large and has its own set of configurations that are generated. We can use the generator to generate matmuls we want to run on Cpu by implementing `CpuMatmulGenerator `when --iree-hal-target-backends is set to cpu.